lib/assessment.rb

```diff
@@ -20,20 +32,32 @@
 # Name and location should be write-only, but age should be read-only.
 
 ##
-# your code here
+class Person
+  attr_accessor :name
+  attr_reader :age
+  attr_accessor :location
```

Comment:
This is not write-only. `attr_accessor` does both read and write (setter and getter).